packages/devtools_app/test/shared/utils_test.dart

```diff
@@ -15,32 +14,30 @@ void main() {
             await Future<void>.delayed(const Duration(seconds: 60));
           },
         );
-        async.elapse(const Duration(seconds: 40));
+        async.elapse(const Duration(milliseconds: 40));
         expect(callbackCounter, 1);
       });
     });
 
-    test('calls the callback once per period', () {
+    test('only triggers another callback after the first is done', () {
       fakeAsync((async) {
         int callbackCounter = 0;
         DebounceTimer.periodic(
           const Duration(seconds: 1),
           (_) async {
             callbackCounter++;
-            await Future<void>.delayed(
-              const Duration(milliseconds: 1),
-            );
+            await Future<void>.delayed(const Duration(seconds: 30));
```

Comment:
are we actually waiting this much time on the test? Make these durations much shorter if so

Reply:
No, these tests use fake async to mock the passage of time. So everything happens almost instantly :)